pkg/generator/status_vhost.go

```diff
@@ -54,7 +54,7 @@ func readyRoute() *route.Route {
 	cluster := envoy.NewWeightedCluster("service_stats", 100, nil)
 	var wrs []*route.WeightedCluster_ClusterWeight
 	wrs = append(wrs, cluster)
-	route := envoy.NewRoute("gateway_ready", nil, "/ready", wrs, 1*time.Second, nil, "")
+	route := envoy.NewRoute("gateway_ready", nil, "/", wrs, 1*time.Second, nil, "")
```

Comment:
Should we use `/probe`, `/kprobe`, `/health` or `/khealth` or something else for both readiness and liveness?
Using the root path should be fine as long as the header matching is used but I am not 100% confident if it is no side effect at all. (e.g. the total number of access to `/` cannot be counted anymore.)